pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -105,6 +106,12 @@ type GitlabInterceptor struct {
 	EventTypes []string   `json:"eventTypes,omitempty"`
 }
 
+// CELInterceptor provides a webhook to intercept and pre-process events
+type CELInterceptor struct {
+	Expression string            `json:"expression,omitempty"`
```

Comment:
maybe we can call this `filterExpression` or `filters`? More explicit than the `Value` field which are also technically CEL expressions!